realm/realm-library/src/androidTestObjectServer/java/io/realm/SyncConfigurationTests.java

```diff
@@ -69,10 +69,10 @@ public void setUp() {
 
     @After
     public void tearDown() {
-        UserStore userStore = SyncManager.getUserStore();
-        for (SyncUser syncUser : userStore.allUsers()) {
-            userStore.remove(syncUser.getIdentity(), syncUser.getAuthenticationUrl().toString());
-        }
+//        UserStore userStore = SyncManager.getUserStore();
```

Comment:
Don't you wan't to add FIXME whenever you comment something out to be able to remember where to uncomment again?

Reply:
Yes, I should probably add FIXME's to all these places

Reply:
I didn't add FIXME comment all those places - but there are quite a few :-)